paper/paper.md

```diff
@@ -0,0 +1,157 @@
+- name: Marta Interlandi
+  orcid: 0000-0002-6863-2552
+  email: marta.interlandi01@gmail.com
+  affiliation: [5]
+- name: Yvan Saeys†
+  orcid: 0000-0002-0415-1506
+  email: yvan.saeys@ugent.be
+  affiliation: [1,2]
+- name: Wouter Saelens†
+  orcid: 0000-0002-7114-6248
+  email: w.saelens@epfl.ch
+  affiliation: [4,10]
+
+affiliations:
+  - id: dambi
+    index: 1
+    department: Data Mining and Modelling for Biomedicine group
+    name: VIB Center for Inflammation Research
+    city: Ghent
+    country: Belgium
+  - id: twist
+    index: 2
+    department: Department of Applied Mathematics, Computer Science, and Statistics
+    name: Ghent University
+    city: Ghent
+    country: Belgium
+  - id: di
+    index: 3
+    name: Data Intuitive
+    city: Lebbeke
+    country: Belgium
+  - id: lmcb
+    index: 4
+    name: Laboratory of Myeloid Cell Biology in tissue homeostasis and regeneration
+    department: VIB Center for Inflammation Research
+    city: Ghent
+    country: Belgium
+  - id: imi
+    index: 5
+    department: Institute of Medical Informatics
+    name: University of Muenster
+    city: Muenster
+    country: Germany
+  - id: nu
+    index: 6
+    name: Northwestern University
+    department: Division of Pulmonary and Critical Care Medicine
+    city: Chicago
+    country: USA
+  - id: icb
+    index: 7
+    name: Helmholtz Munich
+    department: Institute of Computational Biology, Computational Health Center
+    city: Munich
+    country: Germany
+  - id: tum-ma
+    index: 8
+    name: Technical University of Munich
+    department: Department of Mathematics, School of Computation, Information and Technology
+    city: Munich
+    country: Germany
+  - id: lhi
+    index: 9
+    name: Institute of Lung Health and Immunity (LHI), Helmholtz Munich, Comprehensive Pneumology Center (CPC-M), Germany; Member of the German Center for Lung Research (DZL)
+    city: Munich
+    country: Germany
+  - id: dbmb
+    index: 10
+    name: Ghent University
+    department: Department of Biomedical Molecular Biology
+    city: Ghent
+    country: Belgium
+
+engine: knitr
+bibliography: paper.bib
+---
+
+*: Shared first authors, †: Shared last authors.
+
+# Summary
+
+The `{funkyheatmap}` package offers a flexible and user-friendly solution for visualising data frames containing a mixture of categorical, proportional, and text-based data. It simplifies the creation of informative and visually appealing heatmaps while providing extensive customization options to tailor the output. This tool is especially valuable in research settings for summarising and communicating complex results, such as those encountered in benchmarking studies.
+
+The package is available on [CRAN](https://cran.r-project.org/package=funkyheatmap) and [PyPI](https://pypi.org/project/funkyheatmappy/) and has a JavaScript port in development. For detailed examples and vignettes, visit the project website [funkyheatmap.github.io](https://funkyheatmap.github.io).
```

Comment:
I guess that technically speaking the project website no longer offers detailed examples and vignettes but a showcase of examples and an overview of the different implementations. WDYT?

```suggestion
The package is available on [CRAN](https://cran.r-project.org/package=funkyheatmap) and [PyPI](https://pypi.org/project/funkyheatmappy/) and has a JavaScript port in development. For a showcase of examples and an overview of the different implementations, visit the project website [funkyheatmap.github.io](https://funkyheatmap.github.io).
```